utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx

```diff
@@ -33,10 +62,55 @@ export function CardDocumentation() {
           </ul>
         )}
       />
+      <PreCodeForCodeString
+        className="gray-block mt-spacing"
+        codeRaw={`
+          <div class="card p-spacing-xl">
+            <div class="card__title">Title</div>
+            <p>Sed ut perspiciatis unde omnis iste natus error.</p>
+            <div class="flex justify-end">
+              <button type="button" class="button">Contact Us</button>
+              <button 
+                type="button" 
+                class="button button--solid ml-spacing"
+              >
+                See More
+                <span class="button--icon button--icon-right">
+                  <span
+                    class="utds-icon-before-arrow-right"
+                    aria-hidden="true"
+                  ></span>
+                </span>
+              </button>
+            </div>
+          </div>
+        `}
+      />
 
       <StaticExample
         title="Action Cards"
-        renderedExample={<LightBox image={actionCardsScreenshot} alt="Action Cards" className="flex-3up-gap" />}
+        renderedExample={(
+          <div className="flex flex-col gap">
+            <div className="action-card action-card--primary-color action-card--solid">
+              <div className="action-card__title">
+                <h3>Getting Started</h3>
+                <span className="button--icon button--icon-right"><span className="utds-icon-before-arrow-right" aria-hidden="true" /></span>
+              </div>
+              <div className="action-card__body">
+                Learn how to implement the Design System.
+              </div>
+            </div>
+            <div className="action-card action-card--primary-color white-color-background">
+              <div className="action-card__title">
+                <h3>Utah Header and Footer</h3>
+                <span className="button--icon button--icon-right"><span className="utds-icon-before-arrow-right" aria-hidden="true" /></span>
+              </div>
+              <div className="action-card__body">
+                Start using the official header and footer.
+              </div>
+            </div>
+          </div>
+        )}
```

Comment:
The quicktip has a list with a single item which seems like it probably shouldn't be a list? Though the other ones are lists with more than one item... so maybe it's fine?

Reply:
Leave it